### PR TITLE
[RFR] Bootstrap unit tests

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: app/common/fetchMiddleware.spec.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            request: {
+                url: 'http://foo',
+                config: {
+                    foo: 'bar',
+                },
+            },
+            onSuccess,
+        });
+
+        setTimeout(() => {
```
Comment (Kmaschta) [inline app/common/fetchMiddleware.spec.js]: What is the behavior of these `setTimeout`? I don't understand ...

File: package.json [hunk 2]
```diff
@@ -59,6 +59,7 @@
     "winston": "~1.0.2"
   },
   "devDependencies": {
+    "babel-cli": "^6.4.0",
```
Comment (Kmaschta) [inline package.json]: `sed -i 's/"^/"~/g' package.json` in `.git/hooks/precommit` :smile:

File: app/common/fetchMiddleware.spec.js (hunk 1 above)
Comment (djhi) [inline app/common/fetchMiddleware.spec.js]: For a reason I don't get, the callback isn't considered called without it

File: package.json (hunk 2 above)
Comment (djhi) [inline package.json]: damned !

**Pull request merged**